src/coreclr/src/pal/inc/rt/sal.h

```diff
@@ -17,7 +17,7 @@
 
    The comments in this file are intended to give basic understanding of
    the usage of SAL, the Microsoft Source Code Annotation Language.
-   For more details, please see https://go.microsoft.com/fwlink/?LinkID=242134
+   For more details, please see http://go.microsoft.com/fwlink/?LinkID=242134
```

Comment:
why did you revert this one, it doesn't look like a generated file?

Reply:
This file appears to have been copied verbatim from the VC++ SDK. The only thing we added was a header at the top of the file. I didn't want to change any of its contents.